ledger/participant-state/kvutils/tools/src/main/scala/ledger/participant/state/kvutils/tools/integritycheck/StateUpdateComparison.scala

```diff
@@ -44,7 +46,12 @@ final class ReadServiceStateUpdateComparison(
           Future.sequence(
             Seq(
               compareOffsets(expectedOffset, actualOffset),
-              compareUpdates(expectedUpdate, actualUpdate, DefaultNormalizationSettings),
+              compareUpdates(
+                expectedUpdate,
+                actualUpdate,
+                DefaultNormalizationSettings, // TODO: move to normalizers?
```

Comment:
Tomorrow I'd like to move the existing normalization to `UpdateNormalizer`s. One thing that I'm concerned about is that I have to provide an additional config param for expected update normalizers, as they are normalized as well. Let me know what you think and if you are fine with that.